### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by Ravi-t** [pktgen/Pktgen-DPDK#14]
I followed all the instructions mentioned in INSTALL.md but the compilation was failing with error "**../lib/lua/lua_config.c:192:2: error: C++ style comments are not allowed in ISO C90 [-Werror]**"

Thus, I updated the file with C style comment to compile it, please see if you want to add this fix in your release.

./tools/pktgen-build.sh 
>>> lua_enabled      : '-Denable_lua=false'
>>> gui_enabled      : '-Denable_gui=false'
>>> SDK Directory    : '/home/ravi/Pktgen-DPDK'
>>> Build Directory  : '/home/ravi/Pktgen-DPDK/Builddir'
>>> Target Directory : '/home/ravi/Pktgen-DPDK/usr'

[root@cloud Pktgen-DPDK]# ./tools/pktgen-build.sh build
>>> lua_enabled      : '-Denable_lua=false'
>>> gui_enabled      : '-Denable_gui=false'
>>> SDK Directory    : '/home/ravi/Pktgen-DPDK'
>>> Build Directory  : '/home/ravi/Pktgen-DPDK/Builddir'
>>> Target Directory : '/home/ravi/Pktgen-DPDK/usr'

>>> Ninja build in '/home/ravi/Pktgen-DPDK/Builddir' buildtype='release'
meson -Dbuildtype=release -Denable_lua=false -Denable_gui=false Builddir
Directory already configured, exiting Meson. Just run your build command
(e.g. ninja) and Meson will regenerate as necessary. If ninja fails, run ninja
reconfigure to force Meson to regenerate.

If build failures persist, manually wipe your build directory to clear any
stored system data.

To change option values, run meson configure instead.
ninja: Entering directory `/home/ravi/Pktgen-DPDK/Builddir'
[47/75] Compiling C object 'lib/lib@@lua@sta/lua_lua_config.c.o'.
FAILED: lib/lib@@lua@sta/lua_lua_config.c.o 
cc -Ilib/lib@@lua@sta -Ilib -I../lib -Ilib/lua -I../lib/lua -I/usr/local/include -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -Werror -O3 -Wno-pedantic -Wcast-qual -Wdeprecated -Wformat-nonliteral -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wsign-compare -Wstrict-prototypes -Wundef -Wwrite-strings -Wno-missing-field-initializers -include rte_config.h -DALLOW_EXPERIMENTAL_API -pthread -fPIC -include rte_config.h -march=native -mavx -D_GNU_SOURCE  -MD -MQ 'lib/lib@@lua@sta/lua_lua_config.c.o' -MF 'lib/lib@@lua@sta/lua_lua_config.c.o.d' -o 'lib/lib@@lua@sta/lua_lua_config.c.o' -c ../lib/lua/lua_config.c
../lib/lua/lua_config.c:192:2: error: C++ style comments are not allowed in ISO C90 [-Werror]
  // Make sure we display the copyright string for Lua.
  ^
../lib/lua/lua_config.c:192:2: error: (this will be reported only once per input file) [-Werror]
cc1: all warnings being treated as errors
[49/75] Compiling C object 'app/app@@pktgen@exe/pktgen.c.o'.
ninja: build stopped: subcommand failed.

**Comment by pktgen** [comment]
Please use the latest version of DPDK as I do not see which version you are using. Make sure you install DPDK as well.

**Comment by Ravi-t** [comment]
Dear Keith,

Earlier I was compiling it with latest DPDK version 20.11.0-rc0, but then I fall back to 20.08.0 and was able to compile it after making a change in lua_config.c, I have also raised a pull request for it. Please review.

However, after compiling it successfully, I am unable to bring it up for which I have created another issue #16 where even after setting the LD_LIBRARY_PATH, pktgen is complaining that its unable to find librte_timer.so.20.0.

There, I have also copied the output of ldd command which shows that librte_timer.so.20.0 is present, as ldd is showing that all the libraries are available.

**Comment by pktgen** [comment]
Are you using the master branch on github or the ‘dev’ branch? You can try the ‘dev’ branch as it is a bit more up to date then master ATM. With my limit time I only try to test and support the latest DPDK and Pktgen versions. Use DPDK 20.08 and the dev branch of Pktgen which is moving to 20.08 release.

**Comment by Ravi-t** [comment]
Thanks for the information Keith, compilation is successful after removing C++ style comment.
